### PR TITLE
Google code in

### PR DESCRIPTION
**Pull request by abdulsamad234**
Thank you for your pull request! Please read and understand everything below

Please acknowledge that you have read and understood each of the following before submitting your pull request.

# _Do not_ delete any text other than where you are instructed.

**Maintainers: If any of these are left unchecked, do not merge.**

**Students: If one of them is applicable to you -- Please check it anyways. DO NOT REMOVE ANYTHING!**

**You may check them after opening your pull request.**

You may do so by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a live link or preview screenshot
- [x] Images are `240 x 240` [w x h]
- [x] Resolved merge conflicts
- [x] Included a description of my change below

# Things done in this Pull Request

- I added michael christen to the mentors section. The live view is abdulsamad234.github.io/gci16.fossasia.org and my task link is: https://codein.withgoogle.com/dashboard/task-instances/4796878325547008/

### CODE REVIEW

**Comment by Elgayar** [comment]
Can you please provide a live link? or a screenshot.

**Comment by abishekvashok** [comment]
It's there @Elgayar

**Review by abishekvashok — CHANGES_REQUESTED** [review]
Can you add some other details as well search his github, website or other things such as gutter for info or ping him itself

File: _data/mentors.yml [hunk 1]
```diff
@@ -170,3 +170,6 @@
   image: JasonWong.jpg
   twitter: codethejason
   bio: A cryptocurrency enthusiast.
+
+- name: Michael-Christen
+  image: michael.png
```
Comment (robbyoconnor) [inline _data/mentors.yml]: add more details...github, bio,etc.

**Review by robbyoconnor — CHANGES_REQUESTED** [review]

**Comment by Elgayar** [comment]
One more thing, next time, choose a more relevant pull request title..

**Review by abishekvashok — APPROVED** [review]

**Comment by abishekvashok** [comment]
Nice work do you know to squash if you want to learn checkout the tutorials in the contributing.md file pls if not I will merge now itself just tell your opinion

**Comment by abishekvashok** [comment]
@abdulsamad234 pls reply if you could or not mergin for you... Nice work

**Comment by abdulsamad234** [comment]
i don't know how to merge

**Comment by abdulsamad234** [comment]
please help am new to this whole thing @Abhi2424shek

File: _data/mentors.yml [hunk 2]
```diff
@@ -170,3 +170,9 @@
   image: JasonWong.jpg
   twitter: codethejason
   bio: A cryptocurrency enthusiast.
+
+- name: Michael-Christen
+  github: abdulsamad234
```
Comment (robbyoconnor) [inline _data/mentors.yml:175]: Are you kidding me? HIS GITHUB, NOT YOURS!

File: _data/mentors.yml [hunk 3]
```diff
@@ -170,3 +170,9 @@
...
+  image: michael.png
+  twitter: nitrocoder2
```
Comment (robbyoconnor) [inline _data/mentors.yml:177]: His twitter NOT YOURS! How is this not clear?

**Comment by robbyoconnor** [comment]
@abdulsamad234 Can you please explain to me why you added **_YOUR_** Twitter and **_YOUR_** github here? Not only that -- find their actual name, their actual twitter...I've reverted this -- trash your fork and start over.

**Comment by robbyoconnor** [comment]
@abdulsamad234 -- softened up the tone a bit: also you might want to read the [contributing guidelines](https://github.com/fossasia/gci16.fossasia.org/blob/gh-pages/CONTRIBUTING.md) -- which you checked that you read....I get this feeling you just checked them mindlessly -- do not do this! Please put forth some effort, i know you can do it.

File: _data/mentors.yml (hunk 2 above)
Comment (abdulsamad234) [reply _data/mentors.yml:175]: am sorry about that, i could not find his github

File: _data/mentors.yml (hunk 3 above)
Comment (abdulsamad234) [reply _data/mentors.yml:177]: also couldn't

**Comment by abdulsamad234** [comment]
Okay okay, i was just frustrated, i will start all over

**Comment by robbyoconnor** [comment]
@abdulsamad234 -- You were frustrated so you added your github to a mentor's profile? How does that even make sense? 

Yes, you will start over...because I completely reverted it...Literally if you search -- you'll find their github, twitter, blog, etc. I told you how to do it already.

File: _data/mentors.yml (hunk 2 above)
Comment (robbyoconnor) [reply _data/mentors.yml:175]: If you can't find it -- don't add ANY -- NEVER add your own. EVER! 

He has github obviously -- he's a mentor.

Comment (robbyoconnor) [inline _data/mentors.yml]: He has github obviously -- he's a mentor.

**Pull request merged**